Cauli/ViewControllerShakePresenter.swift

```diff
@@ -26,9 +26,12 @@ internal class ViewControllerShakePresenter {
 
     private var viewController: (() -> (UIViewController?))
 
+    /// This will initialize a ViewControllerShakePresenter which will modally present
+    /// a ViewController wrapped within in a UINavigationController.
+    ///
+    /// - Parameter viewController: The viewController wrapped in a UINavigationController.
```

Comment:
That's not true, right? I mean, that isn't a ViewController wrapped in a UINavigationController but one that *will* be wrapped in a UINavigationController, right?

Reply:
🤦‍♂️, yeah. Obviously 🙈